src/main.rs

```diff
@@ -121,6 +121,12 @@ struct Kopium {
     /// Type used to represent maps via additionalProperties
     #[arg(long, value_enum, default_value_t)]
     map_type: MapType,
+
+    /// Always add #[derive(Default)] to structs, even if it can't be derived without a manual impl for some fields
+    ///
+    /// This option only has an effect if `--derive Default` is set.
+    #[arg(long)]
+    force_derive_default: bool,
```

Comment:
I like the idea of having a flag to change behaviour here, but the way it's set up now; `--derive Default --force-derive-default` here feels a bit overkill.

Because this behaviour is a breaking change, maybe we should do this with an opt-in instead, as I suspect that will confuse people a lot less.

An inverted flag could be something like to the effect of "elide impossible derives" like `--smart-derive-elision` / `--partial-derive-detection`. WDYT?

Reply:
Yes, that sounds better to avoid breaking existing scripts. I'll implement your suggestions :+1: